includes/html/forms/copy-dashboard.inc.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
New features should be implemented with laravel, this should be under App\Http\Controllers\Form probably